tests/text/test_tabstops.py

```diff
@@ -34,6 +34,11 @@ def it_knows_its_alignment(self, alignment_get_fixture):
         tab_stop, expected_value = alignment_get_fixture
         assert tab_stop.alignment == expected_value
 
+    def it_can_change_its_alignment(self, alignment_set_fixture):
+        tab_stop, alignment, expected_xml = alignment_set_fixture
+        tab_stop.alignment = alignment
```

Comment:
You'll notice I changed this to `tab_stop.alignment = value`, and similarly for the two other properties.

The reason is that the test doesn't know that the value is an alignment, it can be None in many cases. Plus it keeps the tests as similar (and therefore mechanical and predictable for readers) as possible.